tab-oneipl/minutes/2021_11_29_Minutes.rst

```diff
@@ -0,0 +1,101 @@
+And today on the bridge also we have a lot of people who helped to organize all this activity:
+   - This is IPL managers: Craig Garland who is leading Intel Performance Libraries department, which includes not only Image Processing team, but Math Kernal Library (oneMKL) and several other domains. Svetlana Podogova and Valentin Kubarev - the engineering managers of Intel performance libraries for Image and Signal Processing.
+   - Business Development Group
+   - Technical Consulting Engineers  
+  
+oneAPI industry initiative:
+
+* Nowadays there is a very diverse environment of architectures which are widely used. It includes different types of platforms which are CPUs, GPUs, FPGAs, specific types of accelerators.
+This turns to be a difficult task for developers to support all of these architectures, so they really require some unified environment that allows them to program across all those without deep understanding all of details of each architecture. 
+That is what oneAPI industry initiative is targeted for. 
+
+* The oneAPI is an open unified standard based specification for heterogenous programming which includes DPC++ language spec as a core part, Level Zero spec for the system interface for the oneAPI languages and libraries and a number of elemets covering different domains like maths operations, data processing, video processing and now we are preparing to extend it for Image processing area also. The initial launch of oneAPI Spec v1 in Sept 2020 and since that time the speed of specification development and the number of participants in oneAPI community are actively growing.
+
+* More information about oneAPI may be found in `oneAPI home page <https://www.oneapi.io/>`__ and `oneAPI Spec page <https://www.oneapi.io/spec/>`__.
+
+oneAPI Technical Advisory Board activity:
+
+* The TAB is the place where the future versions of oneAPI specification is discussed. This is invitation-based forum of industry experts, who give their inputs to the oneAPI Specification. It is very critical to hear the input from different companies to adopt the specification and make it unified and meet the challenges of Image Processing area.
+
+* There are 3 other TABs in different domains: the first TAB was created about 2 years ago for DPC++ language and library, then oneMKL (Math Kernal Libraries) TAB started in May 2020, and AI TAB (oneDNN and oneCCL) is active since Feb 2021. More information about TABs may be found in `oneAPI TAB GitHub <https://github.com/oneapi-src/oneAPI-tab>`__
+
+* The goal of oneIPL TAB is to review and collect feedback on oneIPL spec. Make the new specification to address the challenges of Image Processing development and adjust it to the industry needs
+This input will help to shape its next revisions.
+
+* The latest version of oneIPL Spec is published on `oneIPL Spec page <https://spec.oneapi.io/oneipl/latest/index.html>`__
+
+Overview of oneIPL:
```

Comment:
Victor, could you please add some words here

Reply:
Looks ok from the slides